### PR TITLE
Disable expected SILOptimizer test failures.

### PR DESCRIPTION
**Pull request by dan-zheng**
These tests fail because `-enable-ownership-stripping-after-serialization`
has been disabled for `libswiftCore` on `tensorflow` branch due to SR-11336.

TF-799 tracks re-enabling these tests.

### CODE REVIEW

File: test/SILOptimizer/OSLogConstantEvaluableTest.swift [hunk 1]
```diff
@@ -10,9 +10,6 @@
 //
 // REQUIRES: OS=macosx || OS=ios || OS=tvos || OS=watchos
 
-// TODO(TF-799): Re-enable test after SR-11336 is fixed.
```
Comment (dan-zheng) [inline test/SILOptimizer/OSLogConstantEvaluableTest.swift]: Note: this test has been re-enabled because it has started to pass.

**Comment by dan-zheng** [comment]
@swift-ci Please clean test tensorflow

**Review by marcrasi — APPROVED** [review]

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Pull request merged**